experimental/report/testdata/single-line.yaml.fancy.txt

```diff
@@ -2,7 +2,19 @@ remark: "proto4" isn't real, it can't hurt you
   --> foo.proto:1:10
    | 
  1 | syntax = "proto4"
-   |          ^^^^^^^^ help: change this to "proto5"
+   |          ^^^^^^^^^ help: change this to "proto5"
```

Comment:
This change to `^^^^^^^^^` looks one too long.

Reply:
Nope, this change is intentional bc it includes a trailing newline.